src-java/kilda-pce/src/test/java/org/openkilda/pce/impl/AvailableNetworkTest.java

```diff
@@ -35,11 +37,14 @@
 import org.hamcrest.Matchers;
 import org.junit.Test;
 
+import java.util.Collections;
 import java.util.Set;
 import java.util.UUID;
 import java.util.stream.IntStream;
 
 public class AvailableNetworkTest {
+    private static final int cost = 700;
```

Comment:
Constants are usually in upper case, like the weight function below

Reply:
You are right, I will fix it. Thanks!